docs/quickstart.md

```diff
@@ -40,8 +40,70 @@ Note: This process is manual right now, but we're working on providing a Helm ch
 kubectl create -f hack/k8s/manifests/*
 ```
 
-## 7. Creating a sensor
-See the `examples/` directory for a list of sample `Sensors`. Once the `sensor-controller` is deployed, creating a sensor is easy as:
+## 7. Install minio
+
+```
+$ helm init
+...
+$ helm install stable/minio
+...
+
+$ #Verify that the minio pod, the minio service and minio secret are present
+$ kubectl get all -n default -l app=minio
+
+NAME                                        READY     STATUS    RESTARTS   AGE
+pod/laughing-llama-minio-6586bf655c-c5wx2   1/1       Running   0          15h
+
+NAME                           TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
+service/laughing-llama-minio   ClusterIP   None         <none>        9000/TCP   15h
+
+NAME                                   DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/laughing-llama-minio   1         1         1            1           15h
+
+NAME                                              DESIRED   CURRENT   READY     AGE
+replicaset.apps/laughing-llama-minio-6586bf655c   1         1         1         15h
 ```
-kubectl create -f examples/calendar-sensor.yaml
+
+## 8. Create a bucket in Minio and upload the hello-world.yaml into that bucket.
+Download the hello-world.yaml from https://raw.githubusercontent.com/argoproj/argo/master/examples/hello-world.yaml
+```
+$ kubectl port-forward `kubectl get pod -l app=minio -o name` 9000:9000
+```
+Open the browser at http://localhost:9000
+Create a new bucket called 'workflows'.
+Upload the hello-world.yaml into that bucket
+
+## 9. Install Argo
+Follow instructions from https://github.com/argoproj/argo/blob/master/demo.md
+
+## 10. Create an example sensor
+The example sensor below expects a secret called `artifacts-minio` in the default namespace. Change the name from `artifacts-minio` to the actual name of the secret created by minio. In the example below, minion created a secret called `laughing-llama-minio`.
+```
+$ kubectl get secret -l app=minio -n default
+laughing-llama-minio
+
+$ #Replace `artifacts-minio` with `laughing-llama-minio` in the yaml. Then execute the following command.
```

Comment:
If the command to install minio is: `helm install stable/minio --name artifacts --set service.type=LoadBalancer`, then the name of the minio deployment/service/pod/secret will be consistent as `artifacts-minio`. Also by setting the service as LoadBalancer, you don't need to port-forward the port. 

I think specifying the name at least makes things much easier to follow along with, especially since the naming with helm is random every time. The service type is less important to change.

Reply:
+100. Didn't know about the name option in helm. Will do this and update the PR.

On cloud environments (like AWS), the LoadBalancer option results in a publicly accessible LoadBalancer which causes security issues. I'll leave the port-forwarding part as is.